README.md

```diff
@@ -14,3 +14,73 @@ Currently this supports:
 * major releases only
 * all probes registered in separate files (histograms, scalars, events)
 * some select environment data points (more to come)
+
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br>
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br>
+You will also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.<br>
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```

Comment:
Maybe:

> See the *create-react-app documentation about*

In general, these docs look like they were generated by CRA, so it can be a little tricky to understand what they mean.